### PR TITLE
Added stack components

### PR DESCRIPTION
**Pull request by ayush714**
Added stack components column as suggested by Hamza

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,14 +13,13 @@ A collection of `ZenFiles`: Production-grade ML use-cases powered by [ZenML](htt
...
-
+| Name                               | Link                            | Tags                                       | Stack Components                       |
+| ---------------------------------- | ------------------------------- | ------------------------------------------ | -------------------------------------- |
+| Atari Games Reinforcement Learning | [README](atari-game-play)       | reinforcement-learning                     | default                                |
+| Customer Churn                     | [README](customer-churn)        | kubeflow-deployment, seldon-deployment     | kubeflow_orchestrator, seldon_deployer |
+| Customer Satisfaction              | [README](customer-satisfaction) | continuous-deployment                      | mlflow_deployer                        |
+| NBA Predictor                      | [README](nba-pipeline)          | drift, predictions                         | local_kubeflow_orchestrator            |
+| Time Series Forecasting            | [README](time-series-forecast)  | predictions, feature-engineering, vertexai | step_operator, vertexai_stack                     |
```
Comment (htahir1) [inline README.md:22]:
```suggestion
| Time Series Forecasting            | [README](time-series-forecast)  | predictions, feature-engineering, vertexai | step_operator, vertex_stack |
```

File: README.md [hunk 2]
```diff
@@ -13,14 +13,13 @@ A collection of `ZenFiles`: Production-grade ML use-cases powered by [ZenML](htt
...
-| NBA Predictor |  [README](nba-pipeline)   | drift, predictions   |
-| Time Series Forecasting |  [README](time-series-forecast)   | predictions, feature-engineering, vertexai   |
-
+| Name                               | Link                            | Tags                                       | Stack Components                       |
+| ---------------------------------- | ------------------------------- | ------------------------------------------ | -------------------------------------- |
+| Atari Games Reinforcement Learning | [README](atari-game-play)       | reinforcement-learning                     | default                                |
+| Customer Churn                     | [README](customer-churn)        | kubeflow-deployment, seldon-deployment     | kubeflow_orchestrator, seldon_deployer |
+| Customer Satisfaction              | [README](customer-satisfaction) | continuous-deployment                      | mlflow_deployer                        |
+| NBA Predictor                      | [README](nba-pipeline)          | drift, predictions                         | local_kubeflow_orchestrator            |
```
Comment (htahir1) [inline README.md:21]:
```suggestion
| NBA Predictor                      | [README](nba-pipeline)          | drift, predictions                         | kubeflow_orchestrator, evidently            |
```

**Review by htahir1 — CHANGES_REQUESTED** [review]
Thanks! Made some small suggestions. Overall if u accept the suggeestions please format the entire table at the end to be properly formatted in markdown format :-)

**Comment by ayush714** [comment]
Done and I formatted a bit.

**Pull request merged**